theme/less/_body-wrapper.less

```diff
@@ -0,0 +1,21 @@
+body
+{
+  font-family: "PT Sans", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  border-top: 15px solid #375199;
+  margin: 0;
+}
+
+body > div
+{
+  max-width: 960px;
+  margin: 0 auto;
+  padding: 20px 30px;
+}
+
+@media screen and (max-width: 770px)
+{
+  body > div
+  {
+    padding: 20px 50px;
```

Comment:
Suggestion: on mobile left and right paddings are precious, let's change this to at least 20px.